src/managers/visualizer.rs

```diff
@@ -0,0 +1 @@
+// visualizer for emg data - helper file - not a manager
```

Comment:
Let's not check in an empty file -- implementing EMG visualization should be tracked in a separate ticket